node/pkg/processor/vaa.go

```diff
@@ -0,0 +1,47 @@
+package processor
+
+import (
+	"encoding/hex"
+
+	"github.com/certusone/wormhole/node/pkg/vaa"
+	"go.uber.org/zap"
+)
+
+type VAA struct {
+	vaa.VAA
+}
```

Comment:
This is a reasonable tradeoff to make future testing easier.

Reply:
Ack